boards/nordic/nrf54l09pdk/nrf54l09pdk_nrf54l09-common.dtsi

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "nrf54l09pdk_nrf54l09-pinctrl.dtsi"
+
+/ {
+	leds {
+		compatible = "gpio-leds";
+		led1: led_1 {
+			gpios = <&gpio1 7 GPIO_ACTIVE_HIGH>;
+			label = "Green LED 1";
+		};
+		led2: led_2 {
+			gpios = <&gpio0 2 GPIO_ACTIVE_HIGH>;
+			label = "Green LED 2";
+		};
+		led3: led_3 {
+			gpios = <&gpio1 7 GPIO_ACTIVE_HIGH>;
```

Comment:
This GPIO is already assigned to `led1` (copy-paste mistake?): https://github.com/zephyrproject-rtos/zephyr/blob/b6e2e113a138344a213ed7b9252b90dd549e0f42/boards/nordic/nrf54l09pdk/nrf54l09pdk_nrf54l09-common.dtsi#L12-L13

Reply:
Removed the duplicated `led`